designer/client/src/components/toolbars/scenarioActions/buttons/DeployButton.tsx

```diff
@@ -23,6 +25,9 @@ export default function DeployButton(props: ToolbarButtonProps) {
     const capabilities = useSelector(getCapabilities);
     const { disabled, type } = props;
 
+    // TODO: find better place to reload activity capabilities and properties
```

Comment:
What about this todo, will you resolve it in this pr?

Reply:
Fixed: moved into DeployWithParamsDialog.